std/file.d

```diff
@@ -3902,6 +3903,15 @@ string tempDir() @trusted
             DWORD len = GetTempPathW(buf.length, buf.ptr);
             if (len) cache = toUTF8(buf[0 .. len]);
         }
+        else version(Android)
+        {
+            // Don't check for a global temporary directory as
+            // Android doesn't have one.  Instead, use the local
+            // directory or set cache here to your app's local
+            // directory.
+
+            cache = getcwd();
```

Comment:
The user can put this in their app:
```d
string appTempDir()
{
    version(Android)
        return "path/to/appdir";
    else
        return tempDir();
}
```
Then use `appTempDir` instead of `tempDir` in their code. This logic belongs in their program, not in Phobos.